packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -40,6 +41,15 @@ import {
 import { useIsSwitchingL2Chain } from './TransferPanelMainUtils'
 import { NonCanonicalTokensBridgeInfo } from '../../util/fastBridges'
 import { tokenRequiresApprovalOnL2 } from '../../util/L2ApprovalUtils'
+import { userRejectedToast } from '../common/Toast'
+
+const onTxError = (error: any) => {
```

Comment:
should we be more type conscious with Error as go with `error: unknown`, and then do `error as Error`?

Reply:
it is of type `any` in `TransferPanelMain.tsx` so I'm going to keep it as `any` for now until we have better type and checks in place